regulations/check_missing.py

```diff
@@ -0,0 +1,34 @@
+import os
+import subprocess
+
+import re
+MAX_LANG_WIDTH = 20
```

Comment:
Ok, I actually took all the things above from the html.py file ;)
I'll find a way to fix it for both.